lib/usd/ui/layerEditor/abstractCommandHook.h

```diff
@@ -0,0 +1,111 @@
+//
+// Copyright 2020 Autodesk
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#ifndef ABSTRACTCOMMANDHOOK_H
+#define ABSTRACTCOMMANDHOOK_H
+
+#include "pxr/usd/sdf/layer.h"
+
+#include <pxr/usd/usd/stage.h>
+
+#include <QtCore/QString>
+
+#include <string>
+
+namespace UsdLayerEditor {
```

Comment:
Listen, at this moment I've decided the layer editor is not exported from the UI lib and is therefore not in the mayaUsd API and therefore does not need to versionned.  This can be changed in the future, but right now, it doesn't need to be in the public MayaUsd namespace.  The mel commands, however, are.